server/proxies/api-proxy.js

```diff
@@ -2,10 +2,14 @@
 'use strict';
 
 const proxyPath = '/github-issues';
-const target = 'http://localhost:3000';
+const target = 'https://ember-help-wanted-server.herokuapp.com';
```

Comment:
This is a great idea! Can we make this an environment option or something similar? If we want to work on both the server and frontend locally, we'd have to temporarily change this code.